scylla/src/client/session.rs

```diff
@@ -278,6 +281,7 @@ impl SessionConfig {
             timestamp_generator: None,
             keyspaces_to_fetch: Vec::new(),
             fetch_schema_metadata: true,
+            metadata_request_serverside_timeout: Some(Duration::from_secs(2)),
             keepalive_interval: Some(Duration::from_secs(30)),
             keepalive_timeout: Some(Duration::from_secs(30)),
```

Comment:
I wonder if this is the right default value. As far as I understand, the goal is to prevent wasting work (which is wasted, if the cluster is overloaded and we hit the timeout).
Maybe 20 seconds would be better?

cc @mykaul